content/docs/get-started/install/migrating-2.0.md

```diff
@@ -48,6 +48,8 @@ const version: Output<string> = pulumi.output(gcp.container.getEngineVersions())
 
 {{% choosable language python %}}
 
+Optionally, you can use an `async` function [entrypoint]({{< relref "/docs/intro/languages/javascript#optional-async-entrypoint" >}}) to more easily write `async/await` code throughout the rest of your program.
```

Comment:
This may be in the wrong place. Does it belong in the JavaScript section?

Reply:
Yeah... not sure how my eyes saw that as within the JS/TS block. Thanks for catching that!